app/src/test/java/com/mupper/gobus/commons/extension/ContextKtTest.kt

```diff
@@ -0,0 +1,70 @@
+package com.mupper.gobus.commons.extension
+
+import android.content.Context
+import android.content.res.Resources
+import android.graphics.drawable.Drawable
+import com.mupper.gobus.GobusApp
+import com.mupper.gobus.R
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.mock
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.CoreMatchers.instanceOf
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.mockito.BDDMockito.willDoNothing
+import org.mockito.BDDMockito.willReturn
+
+@Suppress("DEPRECATION")
+class ContextKtTest {
+    @Test
+    fun `app should return applicationContext as an instance of GobusApp`() {
+        // GIVEN
+        val mockContext: Context = mock()
```

Comment:
Would it be good moving some mocks like mockContext and mockResources to setUp(Before) function?

Reply:
I think I forget to do the refactor 😅